Harvest.Core/Harvest.Core.csproj

```diff
@@ -1,9 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
I'm really uncomfortable with this change, though I understand why you wanted to do it.  But I'm not sure why I don't like it or if there would be any downsides.  What do you think?

Reply:
That seemed like it was the only way I could get access to `IHttpContextAccessor`. It kept saying something like "Wrong project type" And even then, I was having trouble. Adding `<FrameworkReference Include="Microsoft.AspNetCore.App" />` did the trick.

Reply:
I can't think of any downsides. The only projects referencing it will be under our control and with a compatible version. And any third-parties would be accessing it via API.

Reply:
I had the same issue when I was trying to include the email service. Ended up not doing it.

Reply:
Ok well then we can go with the change, I can't see a scenario in which we would need to use this core lib from a different project framework.  And then we could move the email service in which is useful too.